cmd/envtool/tests_test.go

```diff
@@ -427,34 +233,34 @@ func TestShardTestFuncs(t *testing.T) {
 	t.Run("InvalidIndex", func(t *testing.T) {
 		t.Parallel()
 
-		_, err := shardTestFuncs(0, 3, testFuncs)
+		_, _, err := shardTestFuncs(0, 3, testFuncs)
```

Comment:
Is it possible to test skip shard []string?

Reply:
Yes. Let me think how would be a good way to do so 🤔

Reply:
If you want to test string produced for skip, you could do use `.testdata/`, see `TestListTestFuncs`.